code/controllers/ContentController.php

```diff
@@ -424,12 +414,6 @@ public function successfullyinstalled() {
 		if (!file_exists(BASE_PATH . '/install.php')) {
 			$this->httpError(410);
 		}
-		// The manifest should be built by now, so it's safe to publish the 404 page
```

Comment:
404 and 500 pages are created and published in populateDefaultRecords, so this is redundant as far as I can tell. This bit of code seems to assume a draft 404 page exists and needs publishing but I couldn't find any reference to that anywhere. It only handles the 404 case too, not 500 - so I assume this is a piece of legacy code that got forgotten about?